wire-library/wire-runtime-swift/src/main/swift/CodingUserInfoKey+Wire.swift

```diff
@@ -52,61 +44,98 @@ extension JSONEncoder {
     }
 
     /// The encoding strategy to use for StringEncoded types that are themselves Encodable
-    /// Defaults to .string
+    /// - Note: Defaults to .string
     /// - Note: ProtoMap Dictionary keys are always encoded as strings
+    /// - SeeAlso: [Proto3 JSON Mapping](https://developers.google.com/protocol-buffers/docs/proto3#json)
     public enum StringEncodedEncodingStrategy {
```

Comment:
For a follow up: I'm not totally sure we need this. At least for numbers, the spec is pretty clear that the 64-bit numbers are encoded as strings, and the 32-bits are encoded as numbers. And also that both are allowable when decoding.

Reply:
I'll look into this in my Strip-String-Encodable branch tomorrow

Reply:
Implemented on the Data branch: https://github.com/square/wire/pull/2379/commits/2d20ea1ee39382e81457c4baa6bd666212df84e4